lib/Analysis/Allocation.cpp

```diff
@@ -548,6 +630,9 @@ class AllocationAnalysis {
             xSizeRange.intersects(ySizeRange)) {
           interference[x].insert(y);
         }
+        // if x and y belong to different regions (ignore producer region).
+        if (inDifferentRegion(x, y) && xSizeRange.intersects(yOpRange))
```

Comment:
Oh there is a typo here: xSizeRange.intersects(yOpRange)
-->
xSizeRange.intersects(ySizeRange)

Reply:
While I was looking at that code, I wonder if we should always make `x` and `y` intersect, if they are from different regions.